doc/getting_started/_index.md

```diff
@@ -75,7 +75,7 @@ It should show `~/.local/bin/pip3`.
 If it doesn't, add `~/.local/bin` to your `PATH`, e.g. by adding the following line to your `~/.bashrc` file:
 
 ```console {title=~/.bashrc}
-export PATH=$PATH:~/.local/bin
+export PATH=~/.local/bin:$PATH
```

Comment:
My personal opinion is that I prefer to post append to the path so that system tools are prioritized, especially if we are recommending people add this to their `bashrc` which affects all their terminal sessions.

Reply:
Ah ok, that makes sense to me. It seems my path does have my local bin quite early on.

Using a virtual environment would definitely be better, although our whole getting started guide is set up in a way to install everything system wide. So if we want to have a proper go at doing this for the whole project it would require a more significant rewrite.